### DISCUSSION TITLE
how can i rotation my model around the z axis by mouse move?

### DISCUSSION THREAD

**Discussion by jlygit** [google/model-viewer#1902] [Q&A]
hello all,
how can i rotation my model around the z axis by mouse move?

**Comment by jlygit** [comment]
just like the following
![3d_show](https://user-images.githubusercontent.com/10195989/86361405-60a15400-bca6-11ea-9e08-323e5523c3dc.gif)

**Comment by elalish** [comment]
glTF specifies that the up axis is +Y, so if your model is oriented this way, then you can simply set `camera-controls camera-orbit="0deg 0deg auto" min-camera-orbit="auto 0deg auto" max-camera-orbit="auto 0deg auto"`. phi = 0deg means looking from the top down. We plan to add an orientation attribute in case your model is not Y-up, but until then you'll have to add the transformation to the file.
